contrib/completion/bash/docker

```diff
@@ -5482,10 +5488,14 @@ _docker() {
 	)
 
 	local known_plugin_commands=()
-
-	if [ -f "$COMPOSE_PLUGIN_PATH" ] ; then
-		known_plugin_commands+=("compose")
-	fi
+	local plugin_name=""
```

Comment:
IMHO this code deserves a descriptive comment, e.g. "Create completion functions for all registered plugins"